docker/policies/multiple_entrypoint_instructions_listed_test.rego

```diff
@@ -0,0 +1,80 @@
+package appshield.DS009
+
+test_deny_basic_positive {
+	r := deny with input as {"stages": {
+		"golang": [
+			{
+				"Cmd": "from",
+				"Value": ["golang:1.7.3"],
+			},
+			{
+				"Cmd": "entrypoint",
+				"Value": [
+					"/opt/app/run.sh",
+					"--port",
+					"8080",
+				],
+			},
+			{
+				"Cmd": "entrypoint",
+				"Value": [
+					"/opt/app/run.sh",
+					"--port",
+					"8000",
+				],
+			},
+		],
+		"alpine": [
+			{
+				"Cmd": "from",
+				"Value": ["alpine:latest"],
+			},
+			{
+				"Cmd": "entrypoint",
+				"Value": [
+					"/opt/app/run.sh",
+					"--port",
+					"8080",
+				],
+			},
+		],
+	}}
+
+	count(r) == 2
```

Comment:
This is due to a bug. If you comment out "alpine" stage, you will see the issue. When `fail_entry_points` passes, a non-violating stage ("alpine") will be also detected as well as "golang" by mistake. That's why `count(r) == 2`. It should be `count(r) == 1` since only 1 stage is supposed to be detected.

```suggestion
	count(r) == 1
```

If you compare the result exactly instead of `startswith`, you can identify the bug. That's why I prefer the exact match rather than `startswith`.